perses/tests/test_topology_proposal.py

```diff
@@ -72,6 +74,57 @@ def test_small_molecule_proposals():
         assert smiles == proposal.new_chemical_state_key
         proposal = new_proposal
 
+
+def test_small_molecule_constraint_repair_mapping():
+    """
+    Tests that a constrained edge is demapped after transformed to an unconstrained one.
+
+    This is done by checking that the new_to_old atom map doesn't have the atom that otherwise would be included.
+
+    For the ligands in the test this atom is the ONLY Fluorine atom involved in the transformation.
+    """
+    # TODO: we could try testing a pyridine to a fluoropyridine transformation with using the given geometries
+    from perses.utils.openeye import createOEMolFromSDF
```

Comment:
This could be simplified to
```
# Load molecules
from openff.toolkit.topology import Molecule
off_molecules = Molecule.from_files(sdf_path)
# Change molecule names to `MOL`
[molecule.name = 'MOL' for molecule in off_molecules]
```   
This should get rid of the need for OpenEye molecules entirely (likes 93-105)